components/app/claim/propose/preview/LabelsPreview.tsx

```diff
@@ -1,12 +1,15 @@
-import { EditorStore } from "@/components/app/claim/propose/store/EditorStore";
+import { EditorStore } from "@/components/app/claim/propose/editor/EditorStore";
 import { LabelList } from "@/components/claim/LabelList";
 import { SplitList } from "@/modules/string/SplitList";
 
 export const LabelsPreview = () => {
+  const editor = EditorStore.getState();
+
   return (
     <LabelList
-      labels={SplitList(EditorStore.getState().editor.labels)}
+      labels={SplitList(editor.labels).map(str => str.toLowerCase())}
       lifecycle="propose"
+      target="_blank"
```

Comment:
The labels preview in the claim editor displays the user provided labels in sanitized form and sets the link target to a new browser tab.